scene/resources/mesh.h

```diff
@@ -96,8 +96,9 @@ class Mesh : public Resource {
 		ARRAY_FLAG_USE_2D_VERTICES = ARRAY_COMPRESS_INDEX << 1,
 		ARRAY_FLAG_USE_16_BIT_BONES = ARRAY_COMPRESS_INDEX << 2,
 		ARRAY_FLAG_USE_DYNAMIC_UPDATE = ARRAY_COMPRESS_INDEX << 3,
+		ARRAY_FLAG_USE_OCTAHEDRAL_COMPRESSION = ARRAY_COMPRESS_INDEX << 4,
```

Comment:
This should be added to `_bind_method()` in the `.cpp` (see other constants, and make sure to keep the order).